src/main/java/net/notfab/lindsey/core/commands/fun/Quote.java

```diff
@@ -49,22 +52,27 @@ public CommandDescriptor getInfo() {
 
     @Override
     public boolean execute(Member member, TextChannel channel, String[] args, Message message, Bundle bundle) {
-        try {
-            Request request = new Request.Builder()
-                .url(QUOTES_URL)
-                .addHeader("Accept", "application/json")
-                .addHeader("Content-Type", "application/json")
-                .build();
-
-            Response response = client.newCall(request).execute();
-            String strJson = response.body().string();
-            HashMap<String, String> quote = this.getRandomQuote(strJson);
-            this.sendEmbed(quote, channel);
-
-            return true;
-        } catch (IOException e) {
-            this.handleError(channel);
-            return false;
+        ServerProfile profile = profiles.get(member.getGuild());
+        Language language;
+        if (profile.getLanguage() == null) {
+            language = Language.en_US;
+        } else {
+            language = profile.getLanguage();
+        }
+        List<String> file = getFile(language.name());
+        String strJson = String.join("", file);
+        HashMap<String, String> quote = this.getRandomQuote(strJson);
+        this.sendEmbed(quote, channel);
+
+        return true;
+    }
+
+    private static List<String> getFile(String language) {
```

Comment:
This is a quite heavy operation, executed every time a quote is generated, maybe we should generate it once (including the parsing into a JSONArray) and have it ready in-memory.